src/power_divergence.jl

```diff
@@ -214,6 +235,48 @@ end
 
 # Under regularity conditions, their asymptotic distributions are all the same (Drost 1989)
 # Chi-squared null approximation works best for lambda near 2/3
+"""
+    PowerDivergenceTest(x[, y]; lambda = 1.0, theta0 = ones(length(x))/length(x))
+
+Perform a Power Divergence test.
+
+If `y` is not given and `x` is a matrix with one row or column, or `x` is a vector, then
+a goodness-of-fit test is performed (`x` is treated as a one-dimensional contingency
+table). In this case, the hypothesis tested is whether the population probabilities equal
+those in `theta0`, or are all equal if `theta0` is not given.
+
+If `x` is a matrix with at least two rows and columns, it is taken as a two-dimensional
+contingency table. Otherwise, `x` and `y` must be vectors of the same length. The contingency
+table is calculated using `counts` from [`Statsbase`](@ref). Then the power divergence test
```

Comment:
The `Documenter.jl` documentation states

> Note that depending on what the CurrentModule is set to, a docstring @ref may need to be prefixed by the module which defines it.

However, at least in my local build, links like ``[`StatsBase.counts`](@ref)`` don't seem to work. But maybe we can figure this out in the next PR that will actually set up `Documenter.jl`.